.all-contributorsrc

```diff
@@ -22,6 +22,18 @@
       "contributions": [
         "content"
       ]
+    },
+    {
+      "login": "lobertrand",
+      "name": "Loïc Bertrand",
+      "avatar_url": "https://avatars.githubusercontent.com/u/37326143?v=4",
+      "profile": "https://github.com/lobertrand",
+      "contributions": [
+        "code",
+        "review",
+        "design",
+        "content"
```

Comment:
```suggestion
        "code",
        "content"
```
Reading your comment, I guess you wanted to add me just for code and content, not design and review :smiley:

Reply:
Hah, it doesn't really matter, I just commented with whatever came to mind! Feel free to adjust however you would like to be credited!